README.md

```diff
@@ -17,6 +17,33 @@ The package provides data validation capabilities.
 [![static analysis](https://github.com/yiisoft/validator/workflows/static%20analysis/badge.svg)](https://github.com/yiisoft/validator/actions?query=workflow%3A%22static+analysis%22)
 [![type-coverage](https://shepherd.dev/github/yiisoft/validator/coverage.svg)](https://shepherd.dev/github/yiisoft/validator)
 
+Table of Contents
+=================
+
+* [Yii Validator](#yii-validator)
+    * [Features](#features)
+    * [Requirements](#requirements)
+    * [Installation](#installation)
+    * [General usage](#general-usage)
+        * [Validating a single value](#validating-a-single-value)
+        * [Validating a set of data](#validating-a-set-of-data)
+            * [Skipping validation on error](#skipping-validation-on-error)
+            * [Skipping empty values](#skipping-empty-values)
+            * [Nested and related data](#nested-and-related-data)
+        * [Conditional validation](#conditional-validation)
+        * [Creating your own validation rules](#creating-your-own-validation-rules)
+        * [Grouping multiple validation rules](#grouping-multiple-validation-rules)
+        * [Setting up your own formatter](#setting-up-your-own-formatter)
+    * [Testing](#testing)
+        * [Unit testing](#unit-testing)
+        * [Mutation testing](#mutation-testing)
+        * [Static analysis](#static-analysis)
+    * [License](#license)
+    * [Support the project](#support-the-project)
+    * [Follow updates](#follow-updates)
+
+Created by [gh-md-toc](https://github.com/ekalinin/github-markdown-toc)
```

Comment:
I don't think we need it in README.

Reply:
Sorry, not 100% clear for me.

You mean the whole TOC? Why not? It's getting quite big and becoming harder to navigate. For better sync we can automate it. 

Or you discourage using this specific tool? 

Or mentioning it in `README` (so you mean only the line "Created by `gh-md-toc`..." is redundant)? But I think it's useful - it marks this block as auto generated (we can also add more explicit note) and it contains link to instructions on how to update it.

Reply:
We are eventually going to render these at the website like https://www.yiiframework.com/extension/yiisoft/yii2-redis. There we generate TOC automatically:

<img width="612" alt="image" src="https://user-images.githubusercontent.com/47294/151163327-07b2d34e-bd09-4c4b-8da7-f31b4733438a.png">


"Created by gh-md-toc" despite being technically useful is very discouraging for the user of the library.

Reply:
> We are eventually going to render these at the website

But it's not ready yet. And I think there are still many users reading docs directly from Github, but I don't have stats to prove it.

Anyway, I got it. Removed the whole block.

Reply:
:heavy_check_mark: